### PR TITLE
Added Vendor Title and created SlideIn bar

### PR DESCRIPTION
**Pull request by czhou578**
Added the vendor title and created the slidein bar that will show in the landing page upon toggle.

Close: https://github.com/bcfoodapp/streetfoodlove/issues/222

### CODE REVIEW

**Comment by czhou578** [comment]
Just wanted to see what comments you have about the UI design.

File: app/src/components/UI/Atoms/Map.tsx [hunk 1]
```diff
@@ -67,8 +69,9 @@ export default function Map(): React.ReactElement {
       zoom={14}
       style={{
         height: "92.3vh",
-        width: "100%",
+        width: "180%",
```
Comment (MakotoE) [inline app/src/components/UI/Atoms/Map.tsx:72]: This hides the attribution text.

File: app/src/components/UI/Molecules/LandingPageSidebar/LandingPageSidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        icon
+        onClick={() => setVisible(false)}
+        className={styles.closeIcon}
+      >
+        <Icon name="close" size="big" color="grey" />
+      </Button>
+      <Menu.Item as="div" className={styles.menuItem}>
+        <h3 className={styles.header}>Search</h3>
+        <Container className={styles.search}>
+          <SearchBox />
```
Comment (MakotoE) [inline app/src/components/UI/Molecules/LandingPageSidebar/LandingPageSidebar.tsx:39]: Why is there another search box here when there is one in the header?

**Review by MakotoE — CHANGES_REQUESTED** [review]

File: app/src/components/UI/Atoms/Map.tsx [hunk 3]
```diff
@@ -66,9 +68,10 @@ export default function Map(): React.ReactElement {
       center={[47.584401, -122.14819]}
       zoom={14}
       style={{
-        height: "92.3vh",
-        width: "100%",
+        height: "90.3vh",
+        width: "180%",
```
Comment (MakotoE) [inline app/src/components/UI/Atoms/Map.tsx:72]: The same problem still exists. You should not be setting a percentage relative to the parent element, as the parent element is independent of the window size. The width value should be relative to the window width.
Screenshot from 2022-02-17 10-10-00

File: app/src/components/UI/Pages/LandingPage.tsx [hunk 4]
```diff
@@ -14,9 +17,21 @@ export const LandingPage = () => {
           Version {config.version}, server commit ID {version}
         </p>
       </Container>
-      <Container className={styles.container}>
-        <Map />
-      </Container>
+      <Sidebar.Pushable as={Container} className={styles.container}>
+        <LandingPageSidebar setVisible={setVisible} visible={visible} />
+        <Sidebar.Pusher>
```
Comment (MakotoE) [inline app/src/components/UI/Pages/LandingPage.tsx:22]: I don't think we need a another button to show the sidebar as the search bar button will activate the sidebar.

**Review by MakotoE — CHANGES_REQUESTED** [review]

File: app/src/components/UI/Pages/LandingPage.tsx (hunk 4 above)
Comment (czhou578) [reply app/src/components/UI/Pages/LandingPage.tsx:22]: So the enter button on the search bar will toggle the sidebar on press.... and the map will be able to automatically show the vendors depending on which filters are selected? Just wanted to get this clear.

Comment (MakotoE) [reply app/src/components/UI/Pages/LandingPage.tsx:22]: This is what is written on the SDD:

> If the user decides to enter another destination, they can type in their location in the search bar provided. The results from that search will populate the left sidebar of the map, as well as show on the actual map (with a detailed popup).

So yes, the search results are shown on the sidebar and on the map.

File: app/src/components/UI/Atoms/Map.tsx (hunk 3 above)
Comment (MakotoE) [reply app/src/components/UI/Atoms/Map.tsx:72]: I fixed it like this. Using React devtools, I checked which components were sized smaller than the browser width and set them to max width.

```diff
diff --git a/app/src/components/UI/Atoms/Map.tsx b/app/src/components/UI/Atoms/Map.tsx
index 2bf3d00..af2ecb9 100644
--- a/app/src/components/UI/Atoms/Map.tsx
+++ b/app/src/components/UI/Atoms/Map.tsx
@@ -69,9 +69,8 @@ export default function Map(): React.ReactElement {
       zoom={14}
       style={{
         height: "90.3vh",
-        width: "180%",
+        width: "100%",
         marginTop: 2.9,
-        marginLeft: -425,
         zIndex: 0,
       }}
     >
diff --git a/app/src/components/UI/Pages/LandingPage.tsx b/app/src/components/UI/Pages/LandingPage.tsx
index 4adcc4a..f545c3f 100644
--- a/app/src/components/UI/Pages/LandingPage.tsx
+++ b/app/src/components/UI/Pages/LandingPage.tsx
@@ -20,7 +20,7 @@ export const LandingPage = () => {
       <Sidebar.Pushable as={Container} className={styles.container}>
         <LandingPageSidebar setVisible={setVisible} visible={visible} />
         <Sidebar.Pusher>
-          <Container>
+          <Container fluid>
             <Button
               icon
               className={styles.icon}
diff --git a/app/src/components/UI/Pages/landingpage.module.css b/app/src/components/UI/Pages/landingpage.module.css
index a354ba0..1c75acb 100644
--- a/app/src/components/UI/Pages/landingpage.module.css
+++ b/app/src/components/UI/Pages/landingpage.module.css
@@ -9,7 +9,3 @@
   z-index: 100 !important;
   cursor: pointer;
 }
-
-.container {
-  width: 150% !important;
-}

```

Comment (MakotoE) [reply app/src/components/UI/Atoms/Map.tsx:72]: I believe there is a container that is not set to `fluid`.

File: app/src/components/UI/Pages/LandingPage.tsx (hunk 4 above)
Comment (czhou578) [reply app/src/components/UI/Pages/LandingPage.tsx:22]: I think that having the option to toggle might make navigating the map easier, especially on small screens. I know that users may want to navigate even after results are shown. What do you think?

File: app/src/components/UI/Atoms/Map.tsx (hunk 3 above)
Comment (czhou578) [reply app/src/components/UI/Atoms/Map.tsx:72]: image
Oh wait, I changed this line and it seems to work now!

File: app/src/components/UI/Pages/LandingPage.tsx (hunk 4 above)
Comment (MakotoE) [reply app/src/components/UI/Pages/LandingPage.tsx:22]: On a mobile device, we wouldn't be showing a sidebar as people usually view their screens vertically.

**Pull request merged**